pyproject.toml

```diff
@@ -28,6 +28,8 @@ dependencies = [
     "s3fs>=2024.12.0,<2025",
     "ligo-skymap>=2.2.2",
     "zuko>=1.4.0",
+    "pandas>=2.3.1",
+    "tables>=3.10.1",
```

Comment:
I don't see you import tables, at least in the diff. Where is this needed?

Reply:
Needed for `pandas` hdf support

Reply:
Doesn't pandas provide `pip install pandas[tables]` optional?

Reply:
https://pandas.pydata.org/docs/getting_started/install.html#other-data-sources

It's called hdf5. Let's use that extra.